x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/validate.ts

```diff
@@ -74,7 +75,7 @@ export const transformValidateBulkError = (
   ruleId: string,
   alert: PartialAlert,
   ruleActions?: RuleActions | null,
-  ruleStatus?: unknown
+  ruleStatus?: SavedObjectsFindResponse<IRuleStatusSOAttributes>
```

Comment:
This wasn't typed because some areas were passing through ruleStatuses.saved_objects[0] and some were passing ruleStatuses. Added the correct type and adjusted use cases so they all just pass in ruleStatuses since we were already making checks within for the existence of .saved_objects[0] .

Reply:
Yeah, I think the pattern from Kibana core/other areas go:

* They don't have types or use any or unknown, so let's use unknown for now/cast.
* Later someone adds templates, types, such as `SavedObjectsFindResponse`
* Someone else looks at our code and says, "lol, why you using `unknown` here, let me change that"

So thank you very much. This looks better.